src/_cli_common_utilities.sh

```diff
@@ -3,7 +3,13 @@ function get_user_approval() {
   # If interactive, prompt for user approval & return 0
   # If non-interactive, log failure and return 1
   # If -y/--assume-yes/ASSUME_YES flags/envvars are set, return 0
-  if [ ${ASSUME_YES:-false} == "true" ]; then return 0; fi
+  # If -y/--assume-yes/ASSUME_YES flags/envvars are NOT allowed, skip returning 0
+  # If a parameter is passed when the fucntion is called, its assumed this will ignore -y --yes flags
```

Comment:
```suggestion
  # If a parameter is passed when the function is called, its assumed this will ignore -y --yes flags
```
🙃  (I promise I wasn't here just to look for a spelling error and troll you MB... was just looking for something else lol)

Reply:
lol I'm ripping this out anyway because it will break any automation people have setup to run an update 🙃

Reply:
Awww were you thinking of me? NGL that is part of why I was looking at this PR 😅